### PR TITLE
Add a minify option

### PR DESCRIPTION
**Pull request by Deamon**
To follow up #594 task list,
here is the PR to add the minify option.

### CODE REVIEW

**Comment by alcohol** [comment]
I see packagist only uses it in one place: https://github.com/composer/packagist/blob/ecd1d34940d6bb5ecc932abaae745d8d9e9b0c05/src/Package/V2Dumper.php#L252

Perhaps you could use this as a point of reference?

**Comment by Deamon** [comment]
Thanks for the hint @alcohol! 
With it I was able to find the perfect place for the minify method.

**Comment by alcohol** [comment]
> Thanks for the hint @alcohol!
> With it I was able to find the perfect place for the minify method.
> 
> Do you think I should add
> 
> ```
> "composer/metadata-minifier": "^1.0"
> ```
> 
> to composer.json now we're using it directly in the code ?

You're welcome! And thank you for taking the time to take a look at this issue  :-)

While the package is currently a transient requirement via `composer/composer` (which is one of our direct dependencies), it doesn't hurt to add it to our own list of direct requirements.

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
         run: |
           composer install
           composer --working-dir=tools/php-cs-fixer install
-      - run: composer run php-cs-fixer
+      - run: composer run php-cs-fixer --diff
```
Comment (alcohol) [inline .github/workflows/lint.yaml:32]:
```suggestion
      - run: composer run php-cs-fixer
```

**Comment by Deamon** [comment]
And thank you for your time!

**Pull request merged**